RecoMET/Configuration/python/RecoMET_EventContent_cff.py

```diff
@@ -64,6 +64,7 @@
                                            'drop recoHcalNoiseRBXs_*_*_*',
                                            'keep HcalNoiseSummary_hcalnoise_*_*',
                                            'keep *GlobalHaloData_*_*_*',
+                                           'keep *CSCHaloData_*_*_*',
```

Comment:
Done. I prefer not to touch other lines now but will try to find someone in JetMet that can have a look.